File: Java/Mahasiswa.java
public class Mahasiswa
{
	private String nama;
	private String nim;
	private String prodi;
	private String fakultas;
	
	Mahasiswa(){
	}
	Mahasiswa(String nama,String nim, String prodi, String fakultas){
		this.nama = nama;
		this.nim = nim;
		this.prodi = prodi;
		this.fakultas = fakultas;
	}
	public void setNama(String nama){
		this.nama = nama;
	}
	public String getNama(){
		return this.nama;
	}
	public void setNim(String nim){
		this.nim = nim;
	}
	public String getNim(){
		return this.nim;
	}
	public void setProdi(String prodi){
		this.prodi = prodi;
	}
	public String getProdi(){
		return this.prodi;
	}
	public void setFakultas(String fakultas){
		this.fakultas = fakultas;
	}
	public String getFakultas(){
		return this.fakultas;
	}
	
}

File: php/Mahasiswa.php
<?php

class Mahasiswa
{
	private $nama ='';
	private $nim = '';
	private $prodi = '';
	private $fakultas= '';
	private $foto= '';
	
	public function __construct($nama,$nim, $prodi, $fakultas, $foto){
		$this->nama = $nama;
		$this->nim = $nim;
		$this->prodi = $prodi;
		$this->fakultas = $fakultas;
		$this->foto = $foto;
	}
	public function setNama($nama){
		$this->nama = $nama;
	}
	public function getNama(){
		return $this->nama;
	}
	public function setNim($nim){
		$his->nim = $nim;
	}
	public function getNim(){
		return $this->nim;
	}
	public function setProdi($prodi){
		$this->prodi = $prodi;
	}
	public function getProdi(){
		return $this->prodi;
	}
	public function setFakultas($fakultas){
		$this->fakultas = $fakultas;
	}
	public function getFakultas(){
		return $this->fakultas;
	}
	public function setFoto($foto){
		$this->foto = $foto;
	}
	public function getFoto(){
		return $this->foto;
	}
}

?>